context/app/api/vitessce.py

```diff
@@ -320,8 +320,40 @@ def _build_vitessce_conf(self):
             found_images = _get_matches(file_paths_found, files[0]["rel_path"])
             # Do not show IMS images that are in the "/separate/" folder.
             no_ims_separate_images = _exclude_matches(found_images, r"/separate/")
-            layer = self._build_multi_file_image_layer_conf(no_ims_separate_images)
-            conf["layers"] = [layer]
+            # File paths are like path/to/HybCycle_N/MMStack_PosM.ome.tif and we want
+            # each conf to be all images across a given PosM, each image named HybCycle_N.
+            # This is unique to seqFish.
+            if "seqFish" in self.entity["data_types"]:
+                # Get all PosN names.
+                pos_name_set = set([Path(image).name for image in found_images])
+                # Get all the paths grouped by Pos.
+                images_by_pos = [
+                    [image for image in found_images if Path(image).name == name]
+                    for name in sorted(pos_name_set)
+                ]
+                # Get Hybridization per paths grouped by Pos.
+                hyb_cycles_per_pos = [
+                    sorted([Path(image).parts[-2] for image in images])
+                    for images in images_by_pos
+                ]
+                confs = []
+                # Build up a conf for each Pos.
+                for i, images in enumerate(images_by_pos):
+                    new_conf = copy.deepcopy(conf)
+                    layers = [
+                        self._build_multi_file_image_layer_conf(
+                            images, hyb_cycles_per_pos[i]
+                        )
+                    ]
+                    new_conf["layers"] = layers
+                    # The stem is the name of the ome.tif file (since it has pos) but stem g.
+                    new_conf["name"] = Path(images[0]).stem.split(".")[0]
+                    new_conf = self._replace_view(new_conf)
+                    confs += [new_conf]
```

Comment:
```suggestion
                    confs.append(self._replace_view(new_conf))
```
- Reassigning variables can be a source of confusion.
- `append`.